datahub-graphql-core/src/test/java/com/linkedin/datahub/graphql/resolvers/ingest/secret/CreateSecretResolverMatcherTest.java

```diff
@@ -0,0 +1,45 @@
+package com.linkedin.datahub.graphql.resolvers.ingest.secret;
+
+import com.linkedin.metadata.utils.GenericRecordUtils;
+import com.linkedin.mxe.GenericAspect;
+import com.linkedin.mxe.MetadataChangeProposal;
+import com.linkedin.secret.DataHubSecretValue;
+import org.mockito.ArgumentMatcher;
+
+
+public class CreateSecretResolverMatcherTest implements ArgumentMatcher<MetadataChangeProposal> {
```

Comment:
Awesome glad you're using this!

Reply:
Can we rename this to something more generic? It's really just a matcher for an MCP with a DataHubSecretValue right? I could see this being used elsewhere, especially if we add an UpdateSecret endpoint